packages/core/package.json

```diff
@@ -27,5 +27,11 @@
     "mini-css-extract-plugin": "^2.4.2",
     "webpack": "^5.56.0",
     "webpack-cli": "^4.8.0"
-  }
+  },
+  "peerDependencies": {
+    "@department-of-veterans-affairs/formation": "*",
+    "react": "^16.3.1",
```

Comment:
Do you want to just go ahead and say v17 here since that's what we're [using in vets-website](https://github.com/department-of-veterans-affairs/vets-website/blob/master/package.json#L294)?

Reply:
I can go ahead and do that. We both left comments at the same time for this :upside_down_face: